packages/@ember/-internals/glimmer/tests/integration/helpers/unique-id-test.js

```diff
@@ -1,8 +1,34 @@
-import { RenderingTestCase, strip, moduleFor, runTask } from 'internal-test-helpers';
+import {
+  AbstractTestCase,
+  RenderingTestCase,
+  strip,
+  moduleFor,
+  runTask,
+} from 'internal-test-helpers';
 import { setProperties } from '@ember/object';
+import { uniqueId } from '@ember/helper';
```

Comment:
Here, you'll need to make a couple changes:

1. Use [the `invokeHelper()` utility](https://api.emberjs.com/ember/4.6/functions/@ember%2Fhelper/invokeHelper) to test this behavior.
2. `invokeHelper()` relies on using `getValue` from `@glimmer/tracking/primitives/cache`, and for that you'll need to import from the corresponding internal location.

Net, the updates for imports should look *roughly* like this:

```suggestion
import { uniqueId, invokeHelper } from '@ember/helper';
import { getValue } from '@glimmer/validator';
```